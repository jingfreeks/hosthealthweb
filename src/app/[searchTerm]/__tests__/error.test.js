import { render } from "@testing-library/react";
import SearchTermError from "../error";

describe("Search Term Error Page", () => {
  it("renders Search Term Error Page to match snapshots", () => {
    const { container } = render(<SearchTermError />);
    expect(container).toMatchSnapshot();
  });
});
