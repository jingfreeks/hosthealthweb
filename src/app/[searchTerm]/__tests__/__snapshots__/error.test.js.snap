// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Term Error Page renders Search Term Error Page to match snapshots 1`] = `
<div>
  <main
    class="bg-slate-200 mx-auto max-w-lg py-1 px-4 min-h-screen"
  >
    <h2
      class="my-4 text-2xl font-bold"
    >
      Something went wrong!
    </h2>
    <button
      class="mb-4 p-4 bg-red-500 text-white rounded-xl"
    >
      Try again
    </button>
    <p
      class="text-xl"
    >
      Or go back to
       
      <a
        class="underline"
        href="/"
      >
        Home üè†
      </a>
    </p>
  </main>
</div>
`;
